bridged-tokens-wrapper/src/lib.rs

```diff
@@ -0,0 +1,218 @@
+#![no_std]
+
+use transaction::PaymentsVec;
+
+elrond_wasm::imports!();
+
+#[elrond_wasm::contract]
+pub trait BridgedTokensWrapper {
+    #[init]
+    fn init(
+        &self,
+        universal_bridged_token_id: TokenIdentifier,
+        #[var_args] chain_specific_tokens: MultiValueEncoded<TokenIdentifier>,
+    ) {
+        self.universal_bridged_token_id()
+            .insert(universal_bridged_token_id.clone());
+
+        for token_id in chain_specific_tokens {
+            let _ = self
+                .chain_specific_token_ids(&universal_bridged_token_id)
+                .insert(token_id);
+        }
+    }
+
+    #[only_owner]
+    #[endpoint(addWrappedToken)]
```

Comment:
I don't see a need for this endpoint if you are setting it already on init. This token should remain unchanged